balancer/ui/src/App.tsx

```diff
@@ -1,6 +1,10 @@
 import { Suspense, lazy, useState, useEffect } from "react";
 import { BrowserRouter, Routes, Route } from "react-router-dom";
 
+import { ScoreboardV2Page } from "./pages/v2/ScoreboardV2Page";
+import { TeamDetailPageV2 } from "./pages/v2/TeamDetailPageV2";
+import { ChallengeDetailPageV2 } from "./pages/v2/ChallengeDetailPageV2";
```

Comment:
Please have all components which use motion to be lazy loaded (like the existing score-board)
Motion stuff bloats up the bundle quit a bit. Would prefer to not have it loaded by default.

```tsx
const AdminPage = lazy(() => import("./pages/AdminPage"));
const ScoreOverviewPage = lazy(() => import("./pages/ScoreOverview"));
const IndividualScorePage = lazy(() => import("./pages/IndividualScorePage"));
```